digid_eherkenning/templates/admin/widgets/clearable_private_file_input.html

```diff
@@ -0,0 +1,14 @@
+<!-- Update the django-privates template in order to remove the clear and upload buttons since this field
+is automatically updated. User should only be able to download the file. -->
```

Comment:
this is dangerous! you are now overriding all the widgets everywhere, also if the project itself uses django-privates directly!

Instead, you must define a custom widget class in your `admin.py` and point it to a properly namespaced template.